### PR TITLE
[web-sys] add setConfiguration to RtcPeerConnection

### PR DESCRIPTION
**Pull request by k-wasniowski**
This PR adds `setConfiguration` to RTCPeerConnection object.
https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/setConfiguration

### CODE REVIEW

File: crates/web-sys/webidls/enabled/RTCPeerConnection.webidl [hunk 1]
```diff
@@ -107,7 +107,8 @@ interface RTCPeerConnection : EventTarget  {
   [ChromeOnly]
   attribute DOMString id;
 
-  RTCConfiguration      getConfiguration ();
+  undefined setConfiguration(optional RTCConfiguration configuration);
```
Comment (daxpedda) [inline crates/web-sys/webidls/enabled/RTCPeerConnection.webidl:110]:
```suggestion
  [Throws]
  undefined setConfiguration(optional RTCConfiguration configuration);
```

Comment (daxpedda) [inline crates/web-sys/webidls/enabled/RTCPeerConnection.webidl:110]:
```suggestion
  undefined setConfiguration(optional RTCConfiguration configuration = {});
```
Following the spec es exactly as possible, to make it easier to compare it in diffs later.

**Review by daxpedda — CHANGES_REQUESTED** [review]
Missing a changelog entry.

Otherwise looks good, thank you!

**Comment by k-wasniowski** [comment]
> Missing a changelog entry.
> 
> Otherwise looks good, thank you!

Thank you for your review! Updated changelog and regenerated bindings

**Review by daxpedda — APPROVED** [review]
Thank you!

**Pull request merged**